docker/run_update_data.sh

```diff
@@ -24,9 +24,9 @@ CRONTABBERCMD="./socorro/cron/crontabber_app.py"
 ./docker/run_ftpscraper_wrapper.sh
 
 # Update featured versions data based on release data
-${DC} run app shell bash -c ${CRONTABBERCMD} --reset-job=featured-versions-automatic
-${DC} run app shell bash -c ${CRONTABBERCMD} --job=featured-versions-automatic \
+${DC} run app shell ${CRONTABBERCMD} --reset-job=featured-versions-automatic
+${DC} run app shell ${CRONTABBERCMD} --job=featured-versions-automatic \
```

Comment:
Previously, these would just run crontabber twice because it wasn't passing in the arguments to crontabbercmd. This fixes it so that all the args get passed in so it first resets the featured-versions-automatic app and then runs it.